packages/legend-extension-external-language-morphir/_package.config.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { generateBundleCopyrightText } from '../../scripts/copyright/PackageCopyrightHelper.js';
+import { dirname } from 'path';
+import { fileURLToPath } from 'url';
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+export default {
+  publish: {
+    tsConfigPath: './tsconfig.build.json',
```

Comment:
would need to publish the `package.tsconfig.json` as well. I suggest you refer to something like `packages/legend-extension-dsl-data-space` for these.

Reply:
Fixed!